src/include/miopen/rnn_util.hpp

```diff
@@ -35,6 +35,12 @@
 
 namespace miopen {
 
+enum rnn_direction
+{
+    Forward  = 0,
+    Backward = 1
+};
```

Comment:
It's mostly a question for @atamazov - do we use `enum class` or have plans to switch to `enum class` everywhere?

Reply:
Let's prefer enum class. But switching to it everywhere is impossible IIRC.